components/Organizers.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC } from 'react';
+
+import * as cityData from '../pages/api/data';
+
+export const Organizers: FC = () => {
+  const organizers = Object.values(cityData.data).flatMap(
+    city => city.organiser,
+  );
+
+  return (
+    <div className="content">
+      <p className="h1 mb-4">社区组织者</p>
+      <div className="d-flex flex-wrap">
+        {organizers.map(item => (
+          <div
+            className="list_item mb-5 text-center"
+            key={item?.name}
+            style={{ marginBottom: '50px', width: '20%' }}
+          >
+            {item?.link ? (
+              <a
+                className="text-dark"
+                href={`organiser.html?name=${item.link}`}
+                style={{ fontSize: '15px' }}
+              >
```

Comment:
```suggestion
            {item.link ? (
              <a className="text-dark" href={`organiser/${item.link}`}>
```
1. 静态数据数组里的元素一定存在
2. Next.js 路由没有 HTML 文件扩展名，而且习惯把 ID 性质的参数放在路径中
3. 尽量不用奇数像素值，且尽量取整默认字号 `16px` 的整数、整小数倍